bin/update-analytics-vendors.js

```diff
@@ -0,0 +1,185 @@
+/**
+ * External dependencies
+ */
+const { once } = require( 'events' );
+const fs = require( 'fs' );
+const readline = require( 'readline' );
+const stream = require( 'stream' );
+const { execSync } = require( 'child_process' );
+const axios = require( 'axios' );
+
+/**
+ * File path of the analytics vendors list.
+ */
+const ANALYTICS_VENDORS_FILE = 'includes/data/analytics-vendors-list.php';
+
+class UpdateAnalyticsVendors {
+	/**
+	 * Construct method.
+	 */
+	constructor() {
+		this.vendors = [];
+		this.content = '';
+		this.init();
+	}
+
+	/**
+	 * Initialize the script.
+	 */
+	async init() {
+		await this.fetchData();
+		await this.extractVendors();
+		this.saveData();
+	}
+
+	/**
+	 * Fetch analytics vendors.
+	 *
+	 * @return {Promise<void>}
+	 */
+	async fetchData() {
+		// Remote URL for analytics vendors.
+		const url =
+			'https://raw.githubusercontent.com/ampproject/amphtml/main/extensions/amp-analytics/analytics-vendors-list.md';
+		const response = await axios.get( url );
+
+		// A new readable stream for response data.
+		const bufferStream = new stream.PassThrough();
+		bufferStream.end( response.data );
+
+		// Create a readline interface to read the file line by line.
+		const fileContent = readline.createInterface( {
+			input: bufferStream,
+			crlfDelay: Infinity,
+		} );
+
+		this.content = fileContent;
+	}
+
+	/**
+	 * Extract vendors from recieved data.
+	 *
+	 * @return {Promise<void>}
+	 */
+	async extractVendors() {
+		let commentFlag = false;
+		let vendorSlug = '';
+		let vendorTitle = '';
+
+		try {
+			this.content.on( 'line', ( line ) => {
+				// Check if the line is in a comment.
+				if ( line.trim() === '<!--' ) {
+					commentFlag = true;
+				}
+
+				if ( line.trim() === '-->' ) {
+					commentFlag = false;
+				}
+
+				// Check if line contains vendor title.
+				if ( line.indexOf( '###' ) === 0 && ! commentFlag ) {
+					vendorTitle = line.replace( '###', '' ).trim();
+				}
+
+				// Check if line contains vendor slug.
+				if ( line.indexOf( 'Type attribute value:' ) === 0 && ! commentFlag ) {
+					vendorSlug = line.replace( 'Type attribute value:', '' ).trim();
+				}
+
+				// Populate vendors object.
+				if ( vendorSlug && vendorTitle ) {
+					const vendorSlugs = vendorSlug.replace( /[^\w,\/-]/g, '' ).trim().split( ',' );
+
+					// Loop through multiple vendor slugs with same titles and append extra information to title.
+					vendorSlugs.forEach( ( slug ) => {
+						if ( vendorSlugs.indexOf( slug ) === 0 ) {
+							// If vendor is Google Tag Manager, then add it along with In house analytics for '' value.
+							if ( slug === 'N/A' && vendorTitle === 'Google Tag Manager' ) {
+								slug = '';
+								vendorTitle = 'In house analytics / Google Tag Manager';
+							}
```

Comment:
I think this may be too difficult to find in the list. For the other thing, “In house analytics” should really be translated for the user, so I don't think it should be here. 

What's more is that this UI has never really been suitable for GTM because it requires adding a `config` attribute:

```html
<amp-analytics config="https://www.googletagmanager.com/amp.json?id=GTM-CONTAINER_ID" data-credentials="include"></amp-analytics>
```

Whereas the UI here is all about managing the `type` attribute.

Therefore, I might suggest that instead of having this option, that there be another value added to the beginning/end of the list like “(Other)” and when that is selected, a paragraph with information about in-house analytics and GTM can be added. For GTM specifically, we'd recommend rather that the user use Site Kit instead (which [outputs GTM](https://github.com/google/site-kit-wp/blob/ef0cc0ad2a03c6b6b0ad3f2e72b46902274194fa/includes/Modules/Tag_Manager/AMP_Tag.php#L68-L73)), in the same way we provide this guidance if the user supplies `googleanalytics` as the type.